src/plugins/walletConnectToDapps/components/modals/SessionProposal.tsx

```diff
@@ -147,10 +149,11 @@ const SessionProposal = forwardRef<SessionProposalRef, WalletConnectSessionModal
         Object.entries(optionalNamespaces)
           .map(([key, namespace]): [string, ProposalTypes.BaseRequiredNamespace] => {
             namespace.chains = namespace.chains?.filter(chainId => {
+              const chainAccountIds = selectAccountIdsByChainId(store.getState(), { chainId })
```

Comment:
when `selectAccountIdsByChainId` (non filter version) is added, add to dependency array for reactivity on changes?

Reply:
84414deb62